sdk/bootstrap/main.go

```diff
@@ -130,13 +135,28 @@ func splitAndClean(s string, sep string) []string {
 }
 
 // env print
-
 func printEnv() {
-	env := os.Environ()
+	env := logSensitiveEnviron()
 	sort.Strings(env)
 	log.Printf("Bootstrapping with environment:\n%s", strings.Join(env, "\n"))
 }
 
+func logSensitiveEnviron() (ret []string) {
+	var filteredEnviron []string
+	for _, keyValue := range os.Environ() {
+		// Each string is in format key=value; value may be empty.
+		s := strings.Split(keyValue, "=")
+		key := s[0]
+		value := s[1]
+		if val, ok := blackListedEnvironKeys[key]; ok && len(value) > 0 {
```

Comment:
We should still blacklist even if a key is empty as even this leaks information.